__tests__/routes.test.js

```diff
@@ -24,19 +42,24 @@ describe('GET /api/topics', () => {
 });
 
 // !!! ADD ERROR TESTS LATER !!!
-describe('GET /api', () => {
-    it('provides a description of all other endpoints available', () => {
-        return request(app).get('/api')
+describe('GET /api/articles', () => {
+    it('gets all articles', () => {
+        return request(app).get('/api/articles')
             .expect(200).then((response) => {
-                const endpoints = response.body;
-                // Expect there to be 3 endpoints
-                expect(Object.keys(endpoints)).toHaveLength(3)
-                for (const key in endpoints){
-                    let properties = Object.keys(endpoints[key]);
-                    // Expect the first property of each endpoint to be its description
-                    expect(properties[0]).toBe('description');
-                };
-            });
+            const articles = response.body
+            console.log(articles)
+        });
+    });
+});
+
+// !!! ADD ERROR TESTS LATER !!!
+describe('GET /api/articles/:article_id', () => {
+    it('gets an article by its id', () => {
+        return request(app).get('/api/articles/1')
+            .expect(200).then((response) => {
+            const article = response.body
```

Comment:
we need something like this to test the result thats coming through 
 
  expect(article).toEqual(
          expect.objectContaining({
            author: expect.any(String),
            title: expect.any(String),
            article_id: 1,
            body: expect.any(String),
            topic: expect.any(String),
            created_at: expect.any(String),
            votes: expect.any(Number),
            article_img_url: expect.any(String),
          })
        );